Sources/Parsers/ColorsFileParsers/ColorsTXTFileParser.swift

```diff
@@ -0,0 +1,83 @@
+//
+// SwiftGenKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+final class ColorsTextFileParser: ColorsFileTypeParser {
+  static let extensions = ["txt"]
+
+  private var colors = [String: UInt32]()
+
+  private func addColor(named name: String, value: String) throws {
+    try addColor(named: name, value: parse(hex: value, key: name))
+  }
+
+  private func addColor(named name: String, value: UInt32) {
+    colors[name] = value
+  }
+
+  public func keyValueDict(from path: Path, withSeperator seperator: String = ":") throws -> [String:String] {
+    let content = try path.read(.utf8)
+    let lines = content.components(separatedBy: CharacterSet.newlines)
+    let whitespace = CharacterSet.whitespaces
+    let skippedCharacters = NSMutableCharacterSet()
+    skippedCharacters.formUnion(with: whitespace)
+    skippedCharacters.formUnion(with: skippedCharacters as CharacterSet)
```

Comment:
Hmm, it was added in this commit:
https://github.com/SwiftGen/SwiftGen/commit/ea9c97f20bcd42a38f401b2d2ef1fd1d33f5304b
I can't see why though.

Seems to work fine without, so I'll remove it